packages/components/src/internal/renderers.tsx

```diff
@@ -52,17 +54,21 @@ export function isFilterColumnNameMatch(filter: Filter.IFilter, col: QueryColumn
 interface EditableColumnTitleProps {
     column: QueryColumn;
     editing?: boolean;
+    hideToolTip?: boolean;
     onCancel: () => void;
     onChange: (newValue: string) => void;
 }
 
 // exported for jest tests
 export const EditableColumnTitle: FC<EditableColumnTitleProps> = memo(props => {
-    const { column, editing, onChange, onCancel } = props;
+    const { column, editing, hideToolTip, onChange, onCancel } = props;
     const initialTitle = useMemo(() => {
         return column.caption ?? column.name;
     }, [column.caption, column.name]);
     const [title, setTitle] = useState<string>(initialTitle);
+    const showLabelOverlay: boolean = useMemo(() => {
+        return !hideToolTip && (!!column.description || !!column.format || column.phiProtected);
```

Comment:
I see below that there is a `hasHelpTipData` helper. Can that be used here?

Reply:
Yes, I have that locally. Waiting for merge to finish so I can publish and commit.